hbw/app/javascript/packs/hbw/components/form/select.js.jsx

```diff
@@ -198,23 +198,24 @@ modulejs.define('HBWFormSelect',
       };
 
       fetchOptionsAsync = (inputValue, callback) => {
-        const url = `${this.props.params.url}${(this.props.params.url.includes('?') ? '&' : '?')}`;
-
-        fetch(`${url}q=${inputValue}`)
-          .then(response => response.json())
-          .then((json) => {
-            const res = json.map(opt => ({
-              label: opt.text,
-              value: opt.id
-            }));
+        this.props.env.connection.request({
+          url:    `${this.props.params.url}${(this.props.params.url.includes('?') ? '&' : '?')}`,
```

Comment:
I'm not sure but I think jQuery will handle building url problem completely. Check it please.